praw/models/reddit/live.py

```diff
@@ -8,5 +8,31 @@ class LiveThread(RedditBase):
 
     STR_FIELD = 'id'
 
+    def __eq__(self, other):
+        """Return whther the other instance equals the current."""
+        if isinstance(other, str):
+            return other == str(self)
+        return (isinstance(other, self.__class__) and
+                str(self) == str(other))
+
+    def __hash__(self):
+        """Return the hash of the current instance."""
+        return hash(self.__class__.__name__) ^ hash(str(self))
+
+    def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
+                 _data=None):
+        """Initialize a lazy :class:``LiveThread`` instance.
+
+        :param reddit: An instance of :class:`.Reddit`.
+        :param id: A live thread ID, e.g., ``ukaeu1ik4sw5``
+        """
+        if bool(id) == bool(_data):
+            raise TypeError('Either `id` or `_data` must be provided.')
+        super(LiveThread, self).__init__(reddit, _data)
+        if id:
+            self.id = id  # pylint: disable=invalid-name
+        # else:
```

Comment:
Hmm, I couldn't find the code like `Comment(reddit, _data=...)` but with some grep I found  Reddior class is instantiated with `_data` in [models/user.py](https://github.com/praw-dev/praw/blob/3e1bb8a26d6c5251dc5b7976591e05bc9c690d67/praw/models/user.py#L61):

```python
    def me(self, use_cache=True):  # pylint: disable=invalid-name
        ...
        if self._me is None or not use_cache:
            user_data = self._reddit.get(API_PATH['me'])
            self._me = Redditor(self._reddit, _data=user_data)
return self._me
```